src/Spectre.Angular2Client/src/app/heatmap/heatmap.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { Http, Response, Headers } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import 'rxjs/Rx';
+
+import { Heatmap } from './heatmap';
+
+@Injectable()
+export class HeatmapService {
+
+    private baseUrl = 'http://localhost/spectre_api/api';
```

Comment:
Are we able to put this globally available and in single place? It was repeated several times. Maybe as a global service? This would be useful on future migrations.